.travis.yml

```diff
@@ -23,7 +23,7 @@ before_install:
   - zsh  --version
   - dpkg -s dash | grep ^Version | awk '{print $2}'
 install:
-  - if [ -z "${SHELLCHECK-}" ]; then nvm install node && npm install && npm prune && npm ls urchin doctoc eclint dockerfile_lint; fi
+  - if [ -z "${SHELLCHECK-}" ]; then nvm install --lts && nvm install 16 && npm install && npm prune && npm ls urchin doctoc eclint dockerfile_lint; fi
```

Comment:
```suggestion
  - if [ -z "${SHELLCHECK-}" ]; then nvm install --lts && nvm unalias default && npm install && npm prune && npm ls urchin doctoc eclint dockerfile_lint; fi
```

Reply:
it looks like `nvm install 16 && nvm unalias default` did the trick.  Would you rather have it as `nvm install --lts && nvm unalias default`.
https://app.travis-ci.com/github/nvm-sh/nvm/builds/256448147

Reply:
nah, `--lts` is an imminent timebomb, 16 is a much slower one :-)